e2e/playwright/library-actions/exclude.tests.json

```diff
@@ -1 +1,3 @@
-{}
+{
+    "C286326": "Recent files and Shared tabs are giving error 500 https://alfresco.atlassian.net/browse/ACS-6688"
```

Comment:
@datguychen it's enough to have just link to jira, no need for title of task, could you please change for every test?

Reply:
Fixed

Reply:
thank you